src/components/Dialogs/ActionDialog/ActionDialog.stories.tsx

```diff
@@ -47,30 +49,59 @@ const content = (
   </div>
 )
 
-const RegularTemplate: Story<StoryProps> = ({ showAdditionalAction, ...args }) => {
+const RegularTemplate: Story<StoryProps & any> = ({
```

Comment:
1. I'd rather have us type this properly
2. `StoryProps & any` is just `any`
3. If we must use `any`, we should disable the ESLint warning